raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -27,9 +39,10 @@
     CONTRACT_TOKEN_NETWORK_REGISTRY,
     CONTRACT_USER_DEPOSIT,
 )
+from raiden_contracts.contract_manager import ContractManager
 
-RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT = int(0.075 * 10 ** 18)
-RED_EYES_PER_TOKEN_NETWORK_LIMIT = int(250 * 10 ** 18)
+RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT = TokenAmount(int(0.075 * 10 ** 18))
+RED_EYES_PER_TOKEN_NETWORK_LIMIT = TokenAmount(int(250 * 10 ** 18))
```

Comment:
I ended up adding type annotation to a few test modules, the goal is to catch errors that could have been introduced by my interface changes.